docs/dynamic_batch_size.md

```diff
@@ -0,0 +1,193 @@
+# Dynamic batch size with OpenVINO&trade; Model Server Demultiplexing
+
+## Intoduction
+This documentation presents how to use OpenVINO&trade; Model Server to execute inference requests with dymanic batch size using [Demultiplexing in Directed Acyclic Graph Scheduler](https://github.com/openvinotoolkit/model_server/blob/develop/docs/demultiplexing.md).
```

Comment:
This document guides how to configure DAG Scheduler pipeline to be able to send predict request with arbitrary batch size without model reloading.